assets/js/screens/Collection/List.jsx

```diff
@@ -1,17 +1,27 @@
 import React, { useState, useEffect, useRef } from "react";
 import CollectionListRow from "../../components/Collection/ListRow";
 import { useQuery } from "@apollo/react-hooks";
-import { GET_COLLECTIONS } from "../../components/Collection/collection.query";
+import {
+  GET_COLLECTIONS,
+  DELETE_COLLECTION
+} from "../../components/Collection/collection.query";
 import Error from "../../components/UI/Error";
 import Loading from "../../components/UI/Loading";
-import { Link } from "react-router-dom";
-import debounce from "lodash.debounce";
+import { Link, useHistory } from "react-router-dom";
+import { useMutation } from "@apollo/react-hooks";
+import { useToasts } from "react-toast-notifications";
+
 import Layout from "../Layout";
+import UIModalDelete from "../../components/UI/Modal/Delete";
 
 const ScreensCollectionList = () => {
+  const { addToast } = useToasts();
   const { data, loading, error } = useQuery(GET_COLLECTIONS);
   const [filteredCollections, setFilteredCollections] = useState([]);
-  const [searchValue, setSearchValue] = useState("");
+  const history = useHistory();
+  const [deleteCollectionId, setDeleteCollectionId] = useState(null);
+  const [deleteCollectionName, setDeleteCollectionName] = useState("");
```

Comment:
So this totally works fine, but I'd like to adjust it slightly if you're ok with it.  Instead of having separate `deleteCollectionId` and `deleteCollectionName ` state variables on the component, let's combine the two items into a single convenience object.    The object could be just passing back the entire `collection` object (if it looks reasonably light weight).  If the `collection` object looks like it contains an array of potentially a large number of rows, then probably easier to just define a simple object as argument to the callback function.  Something like?

`{ id: "COLLECTION_ID_HERE", name: "COLLECTION_NAME_HERE" }`

Reply:
So what happens when the user hits "Cancel" on the Delete modal?   Looks like it retains the value of the "active" Collection.  Just to keep the state clean, let's clear out the value of the "active" Collection state variable on "Cancel".  You can just do something like `setActiveCollection(null)` or whatever.